server/pkg/webapp/serve.go

```diff
@@ -139,6 +140,12 @@ func serveConfig(r chi.Router, config webappConfig) {
 			_, _ = fmt.Fprintf(w, line, "SentryDSN", config.sentryUrl)
 		}
 	})
+
+	r.Get(options.CleanBase(config.appUrl, "custom.css"), func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Add("Content-Type", "text/css")
+
+		_, _ = fmt.Fprint(w, service.CurrentSettings.WithDefaults().UI.CustomCSS)
```

Comment:
I'd prefer you bring these in via `serveConfig` args and not referencing an exposed thing